src/DiskCache.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace manifoldco\manifold;
+use manifoldco\manifold\Core;
+
+class DiskCache{
+    static function read_file(){
+        if(!is_readable(storage_path('.manifold.cache.key'))){
+            Core::debug("cache file is not readable");
+            return;
+        }
+        $content = file_get_contents(storage_path('.manifold.cache.key'));
+        $json = json_decode($content);
+        return $json;
+    }
+
+    public static function has($key){
+        $json = static::read_file();
+        if(is_object($json)){
+            if(isset($json->$key)){
+                if(isset($json->$key->expires)){
+                    if($json->$key->expires < time()){
```

Comment:
Shouldn't this be `> time()`?

The reason I am asking is because currently, if the expiration is in the future it will re-fetch the values but if the expiration is in the past the config is empty.

Reply:
@saschat fixed I think now

Reply:
Yes, just tested it and it works!